### PR TITLE
Revise packet creation to integrate with dashboard-based merger

### PR DESCRIPTION
**Pull request by hancush**
## Overview

This PR:

- Updates `_merge_docs` to trigger a `compile_packet` DAG with the relevant values for identifier and attachment links.
- Revises requests for merged documents to hit S3 directly, instead of our proxy file server.
  - Adds a data migration to re-save all packets to update their URLs accordingly.
- Replaces `MERGER_BASE_URL` with two settings: `MERGE_HOST` and `MERGE_ENDPOINT`.

### Checklist

- [x] PR has a descriptive enough title to be useful in changelogs

## Testing Instructions

Performed by @hancush:

 * Manually deploy the app and dashboard to staging, and build/push a staging tag for the app.
 * Shell into the server and save the 50 most recent event packets.
   * 42 of the 50 packets built successfully, with most of the errors being transient 404s we can address by telling scrapelib to retry 404s. Outstanding issues:
     *  https://github.com/datamade/councilmatic-document-merger/issues/1
     * https://github.com/datamade/councilmatic-document-merger/issues/2
     * https://github.com/datamade/councilmatic-document-merger/issues/3
 * Download the generated packets and confirm they look ok.

Handles #738

### CODE REVIEW

File: lametro/models.py [hunk 1]
```diff
@@ -825,8 +828,23 @@ def is_ready(self):
         return self.ready
 
     def _merge_docs(self):
-        merge_url = settings.MERGER_BASE_URL + '/merge_pdfs/' + self.related_entity.slug
-        requests.post(merge_url, json=self.related_files)
+        headers = {
+            'cache-control': 'no-cache',
+            'content-type': 'application/json',
```
Comment (fgregg) [inline lametro/models.py:833]: if you use the `json` argument in `request.post` you should not need to specify the content-type

why do we need cache-control here?

**Review by fgregg — CHANGES_REQUESTED** [review]

File: lametro/models.py (hunk 1 above)
Comment (hancush) [reply lametro/models.py:833]: Not sure. `cache-control` was in the example request in the Airflow docs: https://airflow.apache.org/docs/apache-airflow/stable/deprecated-rest-api-ref.html#endpoints

Comment (fgregg) [reply lametro/models.py:833]: by default, the requests methods don't respect anything related to caching headers, so we can omit that too.

Comment (hancush) [reply lametro/models.py]: Got it, removed the headers entirely in https://github.com/datamade/la-metro-councilmatic/pull/766/commits/13c5726ef17ef7777c46a70d65626738309b3fd3.

**Comment by hancush** [comment]
This has been running on staging for ~10 days. Looks like there have been ~75 merges with only 4 errors: 3 instances of "could not merge encrypted file" (https://github.com/datamade/councilmatic-document-merger/issues/2) and one with an invalid filename (https://github.com/datamade/councilmatic-document-merger/issues/3). Want to get a sense of whether encrypted filenames can be handled in the doc creation workflow before going down that rabbit hole, plan to bring up at next priorities.

**Pull request merged**